src/main/java/com/ecore/roles/model/Membership.java

```diff
@@ -17,7 +14,7 @@
 @Setter
 @Builder
 @Entity
-@Table(uniqueConstraints = @UniqueConstraint(columnNames = {"team_id", "user_id"}))
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = {"role_id", "team_id", "user_id"}))
```

Comment:
Add roleId to the Membership Unique Constraint in order to the User be part of several Role throughout his Teams